### PR TITLE
MIP63 Production Release Changes

### PR DESCRIPTION
**Pull request by hexonaut**
https://forum.makerdao.com/t/mip4c2-spxx-mip63-maker-keeper-network-amendment/19024

### CODE REVIEW

File: MIP63/MIP63.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The only connection is through `dss-vest`, where Keeper Networks are allocated a
 
 #### Keeper Duties and Expectations
 
-It is expected that Keeper Networks will be able to appropriate stream in their budget to be within the requested buffer min and max values. Any excess DAI beyond the max should be returned to the surplus buffer, and the network should act in a timely manor to replenish DAI if the buffer drops below the min value. If the DAI in the buffer consistently is dropping below the min value then a new application should be submitted to increase the budget.
+It is expected that Keeper Networks will be able to appropriate stream in their budget to be within the requested buffer min and max values. Any excess DAI beyond the max should be returned to the surplus buffer, and the network should act in a timely manner to replenish DAI if the buffer drops below the min value. If the DAI in the buffer is consistently running out then a new application should be submitted to increase the budget.
```
Comment (ghost) [inline MIP63/MIP63.md]: I think there should be a maximum duration for these budget requests. I've suggested 6 months but open to other suggestions.

```suggestion
It is expected that Keeper Networks will be able to appropriate stream in their budget to be within the requested buffer min and max values. Any excess DAI beyond the max should be returned to the surplus buffer, and the network should act in a timely manner to replenish DAI if the buffer drops below the min value. If the DAI in the buffer is consistently running out then a new application should be submitted to increase the budget. Budget requests should not exceed a maximum duration of 6 months.
```

Comment (hexonaut) [reply MIP63/MIP63.md]: 6 months is pretty small. Ideally this should eventually be set it and forget it.

Comment (ghost) [reply MIP63/MIP63.md]: Initially I'd written 12 months, happy for you to pick an alternative number. The issue with set it and forget it is that it minimizes opportunity for Governance oversight. The act of requesting a budget forces Governance to assess whether the keeper is providing value or not.

Comment (hexonaut) [reply MIP63/MIP63.md]: Maybe a middle ground we can go with 2 years. This is not the same as a CU where 1 year or less makes sense due to the complex nature of the relationship. Here it is pretty straight forward. They either do their job or not and there is some degree of quality of the work in how fast they execute, how expensive the calls are, etc.

Comment (ghost) [reply MIP63/MIP63.md]: 2 years sounds good to me

Comment (hexonaut) [reply MIP63/MIP63.md]: done

**Comment by blimpa** [comment]
Merged as per [Ratification Poll for MIP63 Maker Keeper Network Amendment (MIP4c2-SP31) - January 9, 2023](https://vote.makerdao.com/polling/QmRswbkm)

**Pull request merged**